portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Comment {
+
+  private final long id;
+  private final String timestamp_UTC;
+  private final String name;
+  private final String email;
+  private final String subject;
+  private final String comments;
+
+  public Comment(long id, String timestamp_UTC, String name, String email, String subject, String comments) {
```

Comment:
Please use camelCase for variable name timestamp_UTC:
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names